devito/symbolics/manipulation.py

```diff
@@ -276,6 +279,25 @@ def _eval_numbers(expr, args):
         args[:] = [expr.func(*numbers)] + others
 
 
+def flatten_args(args, op, ignore=None):
```

Comment:
I know I looked at this recently, can't remember why. I think expr.expand(**hints)` does that but may have limited options